net-misc/curl/curl-8.2.1.ebuild

```diff
@@ -45,19 +46,20 @@ REQUIRED_USE="
 	curl_ssl_nss? ( nss )
 	curl_ssl_openssl? ( openssl )
 	curl_ssl_rustls? ( rustls )
+	nghttp3? ( !openssl )
 "
 
 RDEPEND="
 	sys-libs/zlib[${MULTILIB_USEDEP}]
 	adns? ( net-dns/c-ares:=[${MULTILIB_USEDEP}] )
 	brotli? ( app-arch/brotli:=[${MULTILIB_USEDEP}] )
-	http2? ( >=net-libs/nghttp2-1.15.0:=[${MULTILIB_USEDEP}] )
+	http2? ( >=net-libs/nghttp2-1.12.0:=[${MULTILIB_USEDEP}] )
 	idn? ( net-dns/libidn2:=[static-libs?,${MULTILIB_USEDEP}] )
 	kerberos? ( >=virtual/krb5-0-r1[${MULTILIB_USEDEP}] )
 	ldap? ( net-nds/openldap:=[static-libs?,${MULTILIB_USEDEP}] )
 	nghttp3? (
-		net-libs/nghttp3[${MULTILIB_USEDEP}]
-		net-libs/ngtcp2[ssl,${MULTILIB_USEDEP}]
+		>=net-libs/nghttp3-0.13.0[${MULTILIB_USEDEP}]
+		>=net-libs/ngtcp2-0.17.0[gnutls,ssl,-openssl,${MULTILIB_USEDEP}]
```

Comment:
What's this -openssl thing about?

Reply:
Ah, I see. meh.

Reply:
Honestly it's more to be safe than anything else - as part of this I unmasked `ssl` and `openssl` on `~amd64` as both test fine for net-libs/ngtcp2 (and `ssl,gnutls` is required for curl's TCP/2 support). I don't know what happens if you build CURL against a mixed ngtcp2 binary and frankly I don't want to know right now. Let's make that my problem in a few months when this stuff is all a touch more mature :)